using-mobile-extensions/profile/profile-api-references.md

```diff
@@ -127,14 +131,16 @@ ACPUserProfile.updateUserAttributes(profileMap)
 {% endtab %}
 {% endtabs %}
 
-## **Remove User Attributes**
+## **Remove User Attribute**
```

Comment:
please update to read "Remove a User Attribute

Reply:
updated file with suggetion